src/pages/observations2/index.tsx

```diff
@@ -0,0 +1,29 @@
+import Counter from "@/components/Counter/Counter";
+import { Box, Text, VStack } from '@chakra-ui/react'
+
+const Observations2 = () => {
+    return(
+        <VStack>
+            <Box>
+                <Text as='b'>
+                    Number of Adults 
+                </Text>
+                <Counter type="adults"></Counter>
+            </Box>
+            <Box>
+                <Text as='b'>
+                    Number of Nestlings 
+                </Text>
+                <Counter type="nestlings"></Counter>
+            </Box>
+            <Box>
+                <Text as='b'>
+                    Number of Fledglings 
+                </Text>
+                <Counter type="fledglings"></Counter>
+            </Box>
```

Comment:
I think we can abstract this! I see repeated components, so the layouts are the same, we could just create an array with objects and map over to those components. I will push the update on this as well! 

```jsx
const observations = [
  {
    title: "Number of Adults",
    type: "adults",
  },
  {
    title: "Number of Nestlings",
    type: "nestlings",
  },
  {
    title: "Number of Fledglings",
    type: "fledglings",
  },
];

const Observations2 = () => {
  return (
    <VStack>
      {observations.map((e) => {
        return (
          <Box>
            <Text as="b">{e.title}</Text>
            <Counter type={e.type} />
          </Box>
        );
      })}
    </VStack>
  );
};
```